scss/_buttons.scss

```diff
@@ -137,7 +137,14 @@
 // scss-docs-end btn-variant-loops
 // End mod
 
-// Boosted mod: no .btn-outline
+// Boosted mod: only secondary variant
+.btn-outline-secondary {
+  @include button-variant(transparent, $black, $black, $disabled-background: $white, $disabled-border: $gray-500, $disabled-color: $gray-500);
```

Comment:
I could try but if I do that, the `button-outline-variant` should look like the `button-variant` one since we want hover, active, disabled changes and not only the hover one.

Reply:
Yep that's why I'm not 100% sure it's better in terms of maintenance for us, and usage for advanced and normal users. What do you think?

Reply:
Let's merge it as-is and create an issue to make this enhancement later on.